lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -172,14 +172,15 @@ export class AccountEntity {
      * @param authority
      * @param idToken
      */
-    static createADFSAccount(
+    static createGenericAccount(
         authority: Authority,
         idToken: AuthToken,
-        oboAssertion?: string
+        oboAssertion?: string,
+        authorityType?: AuthorityType
```

Comment:
You can access this property from the `authority` object, don't need to pass it in separately

Reply:
nice catch. changed